gnome/gjs-devel/Portfile

```diff
@@ -26,38 +26,46 @@ distname            ${my_name}-${version}
 dist_subdir         ${my_name}
 use_xz              yes
 
-checksums           rmd160  c4f03a6d3717efba782d2b92c3248e0100d3d011 \
-                    sha256  883b7c2bfe06716f0c44d6a447728f93e7fb810e18a1c48567914bb7d102949e \
-                    size    422548
+checksums           rmd160  168711f499884f0e951dbc47188ec082a95365bf \
+                    sha256  17c0b1ec3f096671ff8bfaba6e4bbf14198c7013c604bfd677a9858da079c0ab \
+                    size    620248
 
 depends_build       port:pkgconfig \
                     port:python310
 
 depends_lib         path:lib/pkgconfig/gtk+-3.0.pc:gtk3 \
                     port:libffi \
                     path:lib/pkgconfig/cairo.pc:cairo \
-                    port:mozjs68 \
+                    port:mozjs91 \
                     port:readline
 
+# GJS 1.72 requires glib 2.66.0
+# When the main glib2 port is updated, change the following
+# line to a path-style dependency
+depends_lib-append  port:glib2-devel
```

Comment:
We'll want to wait until my `glib2`-related PR has been merged, to avoid using a hard dep on `glib2-devel`: https://github.com/macports/macports-ports/pull/15300

Since upstream ports like `gtk3` will pull in `glib2` by default, CI will fail for this port.

Reply:
Since this is the `-devel` port I figured you'd let it slide :-). But yeah if the other PR is close to being merged let's wait.

Reply:
> Since this is the `-devel` port I figured you'd let it slide :-). But yeah if the other PR is close to being merged let's wait.

If Ryan chooses not to move forward with the update to `glib2`, then yep, we can go with `glib2-devel`. But hoping he'll be on-board with that.